### PR TITLE
chore: convert copyright headers to SPDX format

### PR DESCRIPTION
**Pull request by ShGKme**

### CODE REVIEW

File: .reuse/dep5 [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+# L10n
+Files: l10n/*.js l10n/*.json
+Copyright: 2022 Nextcloud GmbH and Nextcloud contributors
+License: AGPL-3.0-or-later
+
+# CI
+Files: .github/**/*
+Copyright: none
+License: CC0-1.0
```
Comment (ShGKme) [inline .reuse/dep5:14]: Not sure if this is correct or can be just `.github/*`, or must be list of the files like on Takl and Notifications

File: .reuse/dep5 [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+# Nextcloud marks
+Files: img/**/*
+Copyright: Nextcloud GmbH <https://nextcloud.com/trademarks/>
+License: LicenseRef-NextcloudTrademarks
+
+# Nextcloud server styles and assets, generated by the server
+Files: src/shared/assets/default/**/*
+Copyright: Nextcloud GmbH and Nextcloud contributors
+License: AGPL-3.0-or-later
```
Comment (ShGKme) [inline .reuse/dep5:44]: Not sure if this is correct. This is styles generated by the server + from the server source

File: AUTHORS.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  - SPDX-License-Identifier: CC0-1.0
+-->
+# Authors
+
+- Grigorii K. Shartsev <me@shgk.me>
+- Joas Schilling <coding@schilljs.com>
+- Julian Petri <julian.petri@iconics.de>
+- Maksim Sukharev <antreesy.web@gmail.com>
+- Richard Steinmetz <richard@steinmetz.cloud>
+- rakekniven <2069590+rakekniven@users.noreply.github.com>
```
[the remark this answers is not in the document]
Comment (nickvergessen) [reply AUTHORS.md:12]: Code, you could have used the script to generate it https://github.com/nextcloud/github_helper/tree/master/spdx-convertor

[the remark this answers is not in the document]
Comment (ShGKme) [reply AUTHORS.md]: Re-checked with converted, fixed 2 wrong years and remade `AUTHORS.md` which misses a lot of contributors now :(

File: .reuse/dep5 (hunk 1 above)
Comment (nickvergessen) [reply .reuse/dep5:14]: after you updated all workflows half of them should be reported as wrong duplicates, currently it's only one:
```
/home/nickv/.local/pipx/venvs/reuse/lib/python3.11/site-packages/reuse/project.py:286: PendingDeprecationWarning: Copyright and licensing information for '.github/workflows/dependabot-approve-merge.yml' has been found in both '.github/workflows/dependabot-approve-merge.yml' and in the DEP5 file located at '.reuse/dep5'. The information for these two sources has been aggregated. In the future this behaviour will change, and you will need to explicitly enable aggregation. See <https://github.com/fsfe/reuse-tool/issues/779>. You need do nothing yet. Run with `--suppress-deprecation` to hide this warning.
  warnings.warn(
```

**Comment by nickvergessen** [comment]
```
The following files have no copyright and licensing information:
* .eslintignore
* .github/dependabot.yml
* .github/pull_request_template.md
* img/talk-icon-mac-shadow.svg
* img/talk-icon-mac.svg
* img/talk-icon-plain-dark.svg
* img/talk-icon-plain-light.svg
* img/talk-icon-rounded.svg
* img/talk-icon-square.svg
* src/shared/assets/default/close.svg
* src/shared/assets/default/icons.css
* src/shared/assets/default/server.css
```

**Comment by ShGKme** [comment]
> The following files have no copyright and licensing information:

> * .eslintignore

Suddenly missed in `.reuse/dep5`

> * .github/dependabot.yml
> * .github/pull_request_template.md

`.reuse/dep5`:

```
# CI
Files: .github/**/*
Copyright: none
License: CC0-1.0
```

Syntax is not correct?

> * img/talk-icon-mac-shadow.svg
> * img/talk-icon-mac.svg
> * img/talk-icon-plain-dark.svg
> * img/talk-icon-plain-light.svg
> * img/talk-icon-rounded.svg
> * img/talk-icon-square.svg

Same

```
# Nextcloud marks
Files: img/**/*
Copyright: Nextcloud GmbH <https://nextcloud.com/trademarks/>
License: LicenseRef-NextcloudTrademarks
```

> * src/shared/assets/default/close.svg
> * src/shared/assets/default/icons.css
> * src/shared/assets/default/server.css

Same:

```
# Nextcloud server styles and assets, generated by the server
Files: src/shared/assets/default/**/*
Copyright: Nextcloud GmbH and Nextcloud contributors
License: AGPL-3.0-or-later
```

**Comment by nickvergessen** [comment]
> Syntax is not correct?

`X/**/*` does not match files inside `X`

I'll send a follow up commit

File: .eslintignore [hunk 4]
```diff
@@ -1 +1,3 @@
+# SPDX-FileCopyrightText: 2023 Nextcloud GmbH and Nextcloud contributors
+# SPDX-License-Identifier: CC0-1.0
```
Comment (ShGKme) [inline .eslintignore:2]: Why it cannot be set in reuse like for other config files?

File: .reuse/dep5 [hunk 5]
```diff
@@ -34,11 +38,11 @@ Copyright: 2017 Feandesign https://soundcloud.com/feandesign
...
 
 # Nextcloud marks
-Files: img/**/*
+Files: img/* img/**/*
 Copyright: Nextcloud GmbH <https://nextcloud.com/trademarks/>
 License: LicenseRef-NextcloudTrademarks
 
 # Nextcloud server styles and assets, generated by the server
-Files: src/shared/assets/default/**/*
+Files: src/shared/assets/default/close.svg src/shared/assets/default/icons.css src/shared/assets/default/server.css src/shared/assets/default/**/*
```
Comment (ShGKme) [inline .reuse/dep5:46]: glob syntax is not allowed?

File: .reuse/dep5 (hunk 2 above)
Comment (ShGKme) [reply .reuse/dep5]: @nickvergessen Question about license is still actual here 👀

File: .eslintignore (hunk 4 above)
Comment (nickvergessen) [reply .eslintignore:2]: We can, but it's "better" to be put in the file directly, especially when it's not shipped.

**Review by nickvergessen — APPROVED** [review]

**Comment by ShGKme** [comment]
Squashed, rebased onto `main`

**Pull request merged**